src/backend/dx11/src/internal.rs

```diff
@@ -53,14 +58,44 @@ struct BlitInfo {
     level: f32,
 }
 
+#[repr(C)]
+struct PartialClearInfo {
+    // transmute between the types, easier than juggling all different kinds of fields..
+    data: [u32; 4],
+}
+
+// the threadgroup count we use in our copy shaders
+const COPY_THREAD_GROUP_X: u32 = 8;
+const COPY_THREAD_GROUP_Y: u32 = 8;
+
+// Holds everything we need for fallback implementations of features that are not in DX. 
+//
+// TODO: maybe get rid of `Clone`? there's _a lot_ of refcounts here and it is used as a singleton
```

Comment:
why is it clonable in the first place?

Reply:
Moved around to a bunch of types. I suppose `Arc` is a good fit here